pandas/io/excel.py

```diff
@@ -97,8 +97,10 @@ def read_excel(io, sheetname=0, **kwds):
         * [0,1,"Sheet5"] -> 1st, 2nd & 5th sheet as a dictionary of DataFrames
         * None -> All sheets as a dictionary of DataFrames
 
-    header : int, default 0
+    header : int, list of ints, default 0
         Row to use for the column labels of the parsed DataFrame
+        If a list of integers is passed those row positions will
```

Comment:
change all of the doc-strings.....FYI, separately (or here if you want). prob should have a common doc-string as this keeps getting repeated all over the place :<